core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -973,6 +973,51 @@ describe('Exploration translation', function() {
     explorationEditorTranslationTab.changeTranslationLanguage('Hindi');
   });
 
+  it('the translation subtab should not change on saving draft', function() {
+    users.createUser('user@translationSubTab.com', 'userTranslationSubTab');
+    users.login('user@translationSubTab.com');
+    workflow.createExploration();
+
+    explorationEditorMainTab.setStateName('one');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'This is first card.'));
+    explorationEditorMainTab.setInteraction('NumericInput');
+    explorationEditorMainTab.addResponse(
+      'NumericInput', forms.toRichText('This is feedback1.'),
+      'two', true, 'Equals', 6);
+    var responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setFeedback(forms.toRichText('This is default_outcome.'));
+    explorationEditorMainTab.addHint('This is hint1.');
+    explorationEditorMainTab.addHint('This is hint2.');
+    explorationEditorMainTab.addSolution('NumericInput', {
+      correctAnswer: 6,
+      explanation: 'This is solution.'
+    });
+    explorationEditorMainTab.moveToState('two');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'This is second card.'));
+    explorationEditorMainTab.setInteraction('NumericInput');
+    explorationEditorMainTab.addResponse(
+      'NumericInput', forms.toRichText('This is feedback1.'),
+      'final card', true, 'Equals', 7);
+    var responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setFeedback(forms.toRichText('This is default_outcome.'));
+    explorationEditorMainTab.addHint('This is hint1.');
+    explorationEditorMainTab.addHint('This is hint2.');
+    explorationEditorMainTab.addSolution('NumericInput', {
+      correctAnswer: 7,
+      explanation: 'This is solution.'
+    });
+    explorationEditorMainTab.moveToState('final card');
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorMainTab.moveToState('two');
+    explorationEditorPage.navigateToTranslationTab();
+    explorationEditorPage.navigateToTranslationTabFeedback();
+    explorationEditorPage.saveChanges();
```

Comment:
Can we also add checks after publishing exploration?

Reply:
Sure, we can!
Adding the required checks.